lib/rule-tester/rule-tester.js

```diff
@@ -578,6 +611,15 @@ class RuleTester {
 
             if (hasOwnProperty(item, "options")) {
                 assert(Array.isArray(item.options), "options must be an array");
+                if (
+                    item.options.length > 0 &&
+                    typeof rule === "object" &&
+                    (
+                        !rule.meta || (rule.meta && typeof rule.meta.schema === "undefined")
```

Comment:
Shouldn't we also check for `null`?

Reply:
@bmish like this?
```suggestion
                        !rule.meta || (rule.meta && (typeof rule.meta.schema === "undefined" || rule.meta.schema === null))
```

Reply:
Yes because both null/undefined indicate the lack of a schema.

Reply:
CI is failing now after adding the check🧐

Reply:
resolved.